modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java

```diff
@@ -130,20 +131,23 @@ protected void doServeResource(
 					"className", _getClassName(contentDashboardItem)
 				).put(
 					"classPK", _getClassPK(contentDashboardItem)
+				).put(
+					"copy", _getCopyJSONObject(contentDashboardItem)
```

Comment:
Why do we need this copy?

Reply:
copy is the name of the FE component we use to copy the file URL
![image](https://user-images.githubusercontent.com/19485114/155299487-56f035b8-280a-4f46-8c68-853b1961f57c.png)

Reply:
But if you think we can use a better name, lest rename it :)